docs/scos/dev/front-end-development/202212.0/oryx/building-components/create-component.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Build Your Custom Oryx Components"
+description: Oryx Components can be replaced by your own components.
+last_updated: Sept 12, 2023
+template: concept-topic-template
+---
+
+This guide helps you to learn how to create a new Oryx component.
+
+Oryx components are web components, built with [Lit](https://lit.dev). Lit is a small framework from Google, optimized for building fast, lightweight web components. Web components can actually be created with any framework or even with vanilla html, CSS and JavaScript. If you do not use Lit, however, you'd miss out on some of the utilities that Oryx provides, such as [signals](docs/scos/dev/front-end-development/{{oage.version}}/oryx/architecture/reactivity/signals.html) and component mixins.
+
+This documentation guide presumes you'll use Lit to build your custom components. It will not go in detail of the standard concepts that are covered in the [component documentation of Lit](https://lit.dev/docs/components/overview/).
+
+In this guide you'll learn how to implement a component and how to use it inside an application.
+
+## Implement a component
+
+In this documentation we'll guide you through the development process to implement a product _ID_ component. It is a simple component that demonstrates a number of basic concepts. The component already exists in the Oryx product package.
+
+Oryx creates a folder per component (e.g. `src/product/id`), and separates some of the component logic in separate files. However, if you are more comfortable with the Single-File Components (SFC) pattern, it is fine to do so. Only the component definition should be separated out to ensure lazy loading of the component.
+
+### Step 1: Create the component class
+
+Oryx components are based on the Web Components standard. One of the features of web components are custom elements. Custom elements are class based elements, that extend from `HTMLElement`. Lit provides the `LitElement` as a base class to extend from when you create a custom element.
+
+```ts
+import { LitElement, TemplateResult } from "lit";
+
+export class ProductIdComponent extends LitElement {
+  protected override render(): TemplateResult {
+    return html`<h1>The product id...</h1>`;
+  }
+}
+```
+
+Oryx components follow a simple naming convention that is used in the class name:
+
+`[Domain][Feature]Component`,
+
+The `Domain` in this example is Product and the feature `Id`.
+
+{% info_block infoBox %}
```

Comment:
formatting leftover?

Reply:
<img width="966" alt="Screenshot 2023-09-20 at 10 38 59" src="https://github.com/spryker/spryker-docs/assets/14858875/cd20cf95-6680-43e8-809c-edefcd386ab5">

Reply:
the usage of info block is on purpose.